src/js/common/schemaform/FormPage.jsx

```diff
@@ -111,14 +125,12 @@ class FormPage extends React.Component {
   }
 
   render() {
-    const { route, params, form } = this.props;
+    const { route, params, form, isLoggedIn } = this.props;
```

Comment:
Is `isLoggedIn` ever true? I can't find where it's passed into this component. You might need to get it from the `user` prop.

Reply:
You're totally right. Thanks, this has been changed.